frontend/fingertips-frontend/components/organisms/Inequalities/inequalitiesHelpers.ts

```diff
@@ -32,24 +32,20 @@ export interface InequalitiesTableRowData {
 export enum Sex {
   MALE = 'Male',
   FEMALE = 'Female',
-  ALL = 'All',
+  ALL = 'Persons',
 }
 
 export enum Inequalities {
   Sex = 'sex',
   Deprivation = 'deprivation',
 }
 
-// 'All' -> 'Persons' mapping to be removed when db value is changed in subsequent ticket
-export const mapToKey = (key: string): string =>
-  key === Sex.ALL ? 'Persons' : key;
-
 export const inequalityKeyMapping: Record<
   Inequalities,
   (keys: string[]) => string[]
 > = {
   [Inequalities.Sex]: (sexKeys: string[]) =>
-    sexKeys.map((key) => mapToKey(key)).toSorted((a, b) => b.localeCompare(a)),
+    sexKeys.map((key) => key).toSorted((a, b) => b.localeCompare(a)),
```

Comment:
I think the map function can be removed 
```suggestion
    sexKeys.toSorted((a, b) => b.localeCompare(a)),
```